interface1.html

```diff
@@ -536,6 +543,12 @@ <h2>Inputs
 			<p>String of terminals: <input type="text" name="inputToGen"> <button id="goButton" type="button">Build syntax</button><button class="orange-button" type="submit" style='margin-left: 5px'>Run GEN</button></p>
 
             <div id="treeUI" style="display: none">
+								<div class="constraint-row">
+									<span><input type="checkbox" name="genOptions" id="trimTrees">Trim Syntactic tree</span>
+									<div class="info">
+										<span class="content">When checked, evaluates mapping constraints against a version of the input tree where silent nodes and terminals of a category other than "x0" or "clitic" are removed. Also removes vaccuous recursion, eg. [[a b]]</span>
```

Comment:
- Cite Elfner 2012. Something like: "in accordance with the prosodic visibility theory proposed in Elfner (2012)"
- vacuous, not vaccuous
- How do nodes get marked as silent?
- Specify that both trees will be shown in the tableau: "The tableau will display both the original and the pruned versions of the syntactic tree."